broader-topics/ecotax.md

```diff
@@ -0,0 +1,59 @@
+# SPECIFICATIONS - ECOTAX
+
+## WHAT IS THE ECOTAX ?
+
+The ecotax is a contribution to finance the collect, the recycling and the depollution of a piece of furniture or used electrical and electronic equipment. 
+It’s paid by the customer. Then it’s donated by the seller to a collection agency. The amount of the ecotax is fixed and does not depend on the product price.
+
+## WHERE IS THE ECOTAX IN PRESTASHOP ?
+
+The ecotax can be enabled in International > Taxes.
+Once enabled, a new drop-down is displayed with all the tax rules displayed and allows the merchant to choose the VAT that will be applied on the ecotax.
+If the ecotax is enabled, then the ecotax field is displayed in the BO add / edit product.
+It shouldn’t be displayed even if enabled if the product is virtual. 
+
+If the ecotax is filled in the back-office for a product, then it's displayed in the front-office product page under the product price: “Including €X.XX for ecotax”. 
+
+## HOW THE ECOTAX WORKS ?
+
+The ecotax is already included in the product price tax excluded. So, the merchant always fills the product price with the ecotax included, and then specify the amount of the ecotax included in the product price. 
+
+The ecotax is provided for information purposes and does not affect the final price because it is already included in it.
+
+The ecotax is subject to VAT, at the rate applicable to the product.
+
+The ecotax can never be reduced. The ecotax must also be applied in the case of products offered.
+
+## Example
+
+Product price tax excl with ecotax: 10€
+
+VAT 20% = 2€
+
+Product price tax incl with ecotax: 12€
+
+
+Ecotax tax excl: 2€
+
+Ecotax VAT 20% = 0,40€
+
+Ecotax tax incl: 2,40€
+
+
+VAT paid by the merchant: 2 + 0,40 = 2,40€
+
+Ecotax paid by the merchant: 2€
+
+Earnings for the merchant: 12 - (2,40 + 2) = 7,60€
+
+### If there is a discount, it should be applied on the product price ecotax excluded
```

Comment:
> Eg. if the price is 10€ including 2€ of ecotax, a 50% rebate will yield a price of 5€ (incl. 2€ of ecotax)

It's not the same result as if the discount is applied on the product price ecotax excluded

10€ including 2€ of ecotax, it's 8€ without ecotax, a 50% rebate will yield a price of 6€ (incl. 2€ of ecotax) => 4€ is the price without ecotax discounted + 2€ of ecotax

> It might be easier to do it on the whole price with a hard limit on the amount of the ecotax.

Yes, but the discount should be applied on the price without ecotax as the ecotax cannot be reduced

Reply:
> discounts must be applied to the price of the product excluding the ecotax

Are you 100% sure about this as a legal requirement? Because since the ecotax is supposed to be already included in the price tax excl, and since rebates are usually calculated based on the final price (including taxes), I'd say that both for the merchant and for the customer it makes sense to do it the way I suggested.

- A - Price tax incl: 10€
- B - Included ecotax: 2€
- C - Rebate: 90%

### My way:

**Final price:** ![max(A \times (1 - C), B) = max(10 \times 0.1, 2) = max(1, 2) = 2](https://render.githubusercontent.com/render/math?math=%5Ctextstyle+max%28A+%5Ctimes+%281+-+C%29%2C+B%29+%3D+max%2810+%5Ctimes+0.1%2C+2%29+%3D+max%281%2C+2%29+%3D+2)


### Your way:

**Final price:** ![C \times (A - B) + B = 0.1 \times (10 - 2) + 2 = 2.8](https://render.githubusercontent.com/render/math?math=%5Ctextstyle+C+%5Ctimes+%28A+-+B%29+%2B+B+%3D+0.1+%5Ctimes+%2810+-+2%29+%2B+2+%3D+2.8)

Reply:
> Are you 100% sure about this as a legal requirement? 

Yes, discounts must be applied to the price of the product and excluding eco-participation as the eco-participation cannot be reduced.

Reply:
Ok verified by accounting. Maybe we should also keep this formula in the spec?

Reply:
Yep, done :heavy_check_mark: